code/modules/admin/view_variables/mark_datum.dm

```diff
@@ -0,0 +1,7 @@
+/client/proc/mark_datum(datum/D)
+	if(!holder)
```

Comment:
See contribution guidelines, all procs should be documented using dmdoc
```suggestion
///Sets the datum D as this clients holders marked datum, given they have a holder
/client/proc/mark_datum(datum/D)
	if(!holder)
```

Reply:
Documentation added. This sufficient?